airbyte-integrations/connector-templates/source-python-http-api/source_{{snakeCase name}}/source.py.hbs

```diff
@@ -114,6 +116,9 @@ class Customers({{properCase name}}Stream):
     TODO: Change class name to match the table/data source this stream corresponds to.
     """
 
+    # TODO: Fill in the primary key. Required. This is usually a unique field in the stream, like an ID or a timestamp.
```

Comment:
The error messages can be quite tough to decipher.  I figured leaving an actual hook here is easier.